### ISSUE TITLE
Singapura's Clip.town returns sensor_id

### ISSUE THREAD

**Issue by pzinemanas** [soundata/soundata#103]
Hi all!

The `town` property of Singapura's Clip is returning the `sensor_id` instead ([see](https://github.com/soundata/soundata/blob/main/soundata/datasets/singapura.py#L259)). This should be very easy to fix:

```
  @property
  def town(self) -> str:
      """
      The clip's location
      Returns:
          * str - location of the sensor, one of {'East 1', 'East 2', 'West 1', 'West 2'}
      """
      return self._clip_metadata["town"]
``` 

However, I wonder how it passed the `test_clip` test since it is supposedly checking town property.

**Comment by kwatcharasupat** [comment]
Hi, @pzinemanas. Thanks for raising this up! I will take a look at it and send in a PR to fix this.

Edit: Interestingly, this bug wasn't caught in the test, even though we wrote an attribute check for it.
https://github.com/soundata/soundata/blob/0659f7d95f84d7a9049af0238aba3d12d4f1d867/tests/datasets/test_singapura.py#L11-L49

cc: @magdalenafuentes, @bhanlam, @kenowr

**Comment by kwatcharasupat** [comment]
I did some digging and found the culprit. Essentially, the if/elif clause as is will cause the expected values in `expected_attributes` to be ignored if the attribute key is already present in `expected_property_types`. 

https://github.com/soundata/soundata/blob/0659f7d95f84d7a9049af0238aba3d12d4f1d867/tests/test_utils.py#L26-L29

I will raise a PR to fix this as well.

**Comment by magdalenafuentes** [comment]
Closed in #104
